modules/http/Makefile.am

```diff
@@ -18,7 +18,9 @@ modules_http_libhttp_la_SOURCES = \
   modules/http/http-parser.c        \
   modules/http/http-parser.h        \
   modules/http/http-plugin.c        \
-  modules/http/http-signals.h
+  modules/http/http-signals.h       \
+  modules/http/http-curl-compression.c \
+  modules/http/http-curl-compression.h
```

Comment:
These belong to an earlier commit: `http: Added compression files and skeleton`

Reply:
Originally wanted to name this commit "Added automake support". Will fix in next rebase.

Reply:
Done